projects/ccf-eui/src/app/app-web-component.component.ts

```diff
@@ -62,6 +63,8 @@ export class AppWebComponent extends BaseWebComponent {
         dataSources: parseDataSources,
         useRemoteApi: BUILTIN_PARSERS.boolean,
         header: BUILTIN_PARSERS.boolean,
+        //using parseDataSources since both have same types
+        selectedOrgans: parseDataSources,
```

Comment:
Rename parse function to `parseStringArray`. Also update the error message in the thrown error